common/src/main/java/org/jboss/hal/testsuite/util/ElytronIntegrationChecker.java

```diff
@@ -190,6 +194,46 @@ private Address addCredentialStore(String credentialStoreName) throws IOExceptio
         return credentialReferenceAddress;
     }
 
+    private void addAlias(final String credentialStoreAliasValue, final Address credentialReferenceAddress)
+            throws IOException {
+        if (ConfigUtils.isDomain()) {
+            for (String server : allRunningServersOnDefaultHost()) {
+                Address serverCredentialReferenceAddress = Address.host(ConfigUtils.getDefaultHost())
+                        .and(Constants.SERVER, server).and(SUBSYSTEM, ELYTRON)
+                        .and(CREDENTIAL_STORE, credentialReferenceAddress.getLastPairValue());
+                invokeAddAlias(credentialStoreAliasValue, serverCredentialReferenceAddress);
+            }
+        } else {
+            invokeAddAlias(credentialStoreAliasValue, credentialReferenceAddress);
+        }
+    }
+
+    private List<String> allRunningServersOnDefaultHost() throws IOException {
+        Address hostAddress = Address.host(ConfigUtils.getDefaultHost());
+        ModelNodeResult result = operations.readChildrenNames(hostAddress, Constants.SERVER);
+        result.assertDefinedValue();
+        List<String> servers = result.stringListValue();
+
+        List<String> startedServers = new ArrayList<String>();
```

Comment:
Creaper has methods like org.wildfly.extras.creaper.core.online.operations.admin.DomainAdministration#allRunningServers(java.lang.String), why don't you make use of it?